### ISSUE TITLE
Enter key handler on Select elements

### ISSUE THREAD

**Issue by nickfederighi** [vuejs/vue#4660]
### Vue.js version
2.1.8

### Reproduction Link
https://jsfiddle.net/nickfederighi/o8fwf0gh/5/

### Steps to reproduce
Assign an event handler to a select element to override the default behavior of opening the element. See below: 
`<select @keyup.enter.prevent="keyHandler">`
### What is Expected?
I would expect the keyHandler method to fire when the select element has focus and the 'enter' key is pressed
### What is actually happening?
Enter key presses on select elements are not handled by the listener. All other keys appear to work fine. See my jsfiddle for an easy reproduction. Set focus on the select element and press any keyboard key and the handler will be called. Pressing the enter key has no impact. 

The default behavior on browsers is for the enter key to expand the select list, but I would like it to submit the form data to be consistent with other inputs on the form that have the same behavior. Adding the 'prevent' modifier to the  inline handler doesn't appear to override this behavior.

**Comment by LinusBorg** [comment]
What browser did you test this with? On OSX Sierra, Chrome 55 it seems to work:

![kapture 2017-01-05 at 16 23 47](https://cloud.githubusercontent.com/assets/1444526/21685833/59d9a83c-d363-11e6-9e4e-20730f7a7803.gif)

**Comment by nickfederighi** [comment]
Fascinating. I'm on Windows 10, Chrome 55

**Comment by nickfederighi** [comment]
I can confirm that on Firefox 50.1.0 on Windows that the behavior is as expected. This appears to be a Windows Chrome issue.

**Comment by LinusBorg** [comment]
Does anyone have an idea about how to rework around this Chrome bug? If not, I will close this issue because it's not fixable in Vue.

**Comment by posva** [comment]
@LinusBorg I agree with you.
@nickfederighi You should look up https://bugs.chromium.org/p/chromium/issues/list to see if the bug report already exists and report it if not

**Comment by nickfederighi** [comment]
Thanks for the help on this, guys, but I believe I've solved the problem on Chrome for Windows. There seems to be a difference between browsers as well as  Windows vs OSX on chrome as to whether the default select behavior is triggered by keyup or keydown. Changing the handler to `@keydown` allowed me to intercept the default behavior. 

Fixed jsFiddle:
https://jsfiddle.net/nickfederighi/o8fwf0gh/7/
